### PR TITLE
Backport bugfixes to 0.10

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

**Comment by laanwj** [comment]
Concept ACK

**Comment by luke-jr** [comment]
Ok, Travis workaround backported and passing now.

File: contrib/debian/changelog [hunk 1]
```diff
@@ -178,7 +178,7 @@ bitcoin (0.5.0~rc1-natty1) natty; urgency=low
 
   * Add test_bitcoin to build test
   * Fix clean
-  * Remove uneccessary build-dependancies
+  * Remove unnecessary build-dependancies
```
Comment (laanwj) [inline contrib/debian/changelog]: dependancies, huh :-)
This is just a backport, but I wonder why the other word spelling wasn't fixed.

**Comment by laanwj** [comment]
ACK.
Going to merge this to make Travis pass again.

**Pull request merged**